src/pybind/mgr/rgw/module.py

```diff
@@ -4,28 +4,55 @@
 import errno
 import base64
 import functools
+import sys
 
-from mgr_module import MgrModule, CLICommand, HandleCommandResult
+from mgr_module import MgrModule, CLICommand, HandleCommandResult, Option
 import orchestrator
 
-from ceph.deployment.service_spec import RGWSpec, PlacementSpec, SpecValidationError
-from typing import Any, Optional, Sequence, Iterator, List
+from ceph.deployment.service_spec import RGWSpec, PlacementSpec, SpecValidationError  # type: ignore
```

Comment:
interesting, which part of this was causing issues that required the `type: ignore`?  I know all of these things get imported somewhere within mgr/cephadm and I don't remember having any of the imports getting marked like this.